daskhub-rhg/values-dev.yaml

```diff
@@ -0,0 +1,59 @@
+# Helm chart values for the prod environment
```

Comment:
should we separate out users into its own yml file (or part of values.yml?) Seems like these should probably be the same across dev and prod? Or do you see reasons for keeping them differentiable across dev and prod

Reply:
If they're shared then could we put them into `values.yaml`?

For every new "values" file we add for the helm chart, we need to add another line when we create the app in `argocd`.

Reply:
yeah I'm fine with putting them in values.yaml. I'm also fine with having one extra line that is --values users.yaml and maintaining an extra file. I kind of think of the users definition as separate than the other config stuff so made some organizational sense to me to maintain a separate file. But doesn't really matter in the end and I'm fine w/ whatever you think makes sense

Reply:
Okay. I broke users back out into their own values file in `642ee95b934d1cf6dca88b9268a1cda0a544f099`. Also updated the README to add the extra values line.